Modules/+Stages/NanoMax300.m

```diff
@@ -0,0 +1,2 @@
+classdef NanoMax300 < Modules.Stage
```

Comment:
Is this just a placeholder class, or what is this?

Reply:
This might have been created by mistake. We'll consider deleting it

Reply:
This also reminds me we should consider adding a warning message when you try to open up the APT stage that is now deprecated.